azurerm/internal/services/digitaltwins/digital_twins_endpoint_eventhub_resource_test.go

```diff
@@ -0,0 +1,304 @@
+package digitaltwins_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/digitaltwins/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type DigitalTwinsEndpointEventHubResource struct{}
+
+func TestAccAzureRMDigitalTwinsEndpointEventHub_basic(t *testing.T) {
```

Comment:
Sorry, I missed this on my first review... can we change this for **all** tests?
```suggestion
func TestAccDigitalTwinsEndpointEventHub_basic(t *testing.T) {
```